pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -0,0 +1,102 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Amazon EKS for Enterprise Early Access
```

Comment:
The pricing table looks nice, but it ends up quite long, only to really tell the story that there are two features different for early-access: the private shared Slack channel and SLAs on response times...

We may want to:

1. Shorten the number of features. E.g., "Reference Applications" can be one item, with "Spring Boot" and "Ruby on Rail" listed as examples in the mouse-over text, rather than each one as a separate row. Similarly, "CI / CD" can be one item that captures Argo & Flux; "Service Mesh" can be one item that captures Istio and Consul Connect, etc.
2. Are there any other advantages we can offer early access subscribers?

Reply:
> Shorten the number of features

Done.

> Are there any other advantages we can offer early access subscribers?

I threw in patcher here (see https://github.com/gruntwork-io/gruntwork-io.github.io/pull/518#discussion_r684539522). I think it might also be worth calling out that early access subscribers can influence which technologies we support out of the list (which is true - that is why the form includes the check box), but not sure if we wanted to mention that.